### DISCUSSION TITLE
Peak and off peak prices

### DISCUSSION THREAD

**Discussion by PJMaud** [tnmendes/watt#104] [Ideas]
It would great if you could add separate prices and times for low price electric. Economy 7 Octopus Go or Intelligent type tariffs

**Comment by tnmendes** [comment]
Hey @PJMaud, Thanks for the suggestion 👍

**Reply by PJMaud** [reply]
No worries. Not related but I’m not sure how to refresh the devices when I add a device or change the name? I can only logout and back in which means I have to create all my rooms again.

**Reply by tnmendes** [reply]
Hey, right now you need to complete close the app, when you open again it will reload all your devices. 
If you also reboot the iPhone will also work.
